regression-test/suites/statistics/test_schema_change_statistics.groovy

```diff
@@ -100,45 +100,37 @@ suite("test_schema_change_statistics") {
     result = sql """show column cached stats change"""
     assertEquals(8, result.size())
 
-    sql """ALTER TABLE change DROP COLUMN mv_value1 from mv1;"""
-    wait_schema_change_finished()
```

Comment:
why remove alter table?

Reply:
we don't support modify mv's column directly.